### ISSUE TITLE
request for additional waypoint support

### ISSUE THREAD

**Issue by drtedastro** [MuMech/MechJeb2#1136]
## MechJeb Version
864
## KSP Version
1.6.1 build 2401
## Description
Wanting to add waypoints to be used by aircraft in the aircraft autopilot and possibly in the approach and autoland.  Ability to create, store and use individual waypoints and the building or routes (along same idea as the rover has).  Basically the merge of functions of the autopilot and approach autoland.  

## Replication Case
no problem, just request

## KSP.log
[KSP.log](https://github.com/MuMech/MechJeb2/files/2872442/KSP.log)

**Comment by drtedastro** [comment]
just checking to see if anyone has even looked at this...

just asking for ability to have more waypoints  than currently  is allowed...

**Comment by drtedastro** [comment]
opps wrong button.

**Comment by drtedastro** [comment]
just wanting to be able to add more waypoints...

**Comment by lamont-granquist** [comment]
uh kind of a dup of #522 ?

**Comment by drtedastro** [comment]
I don't think so.  I am just wanting to be able to add more sites and runways to the LandingSites.cfg.
